template/scala-akka-http-server/controller.mustache

```diff
@@ -99,20 +98,20 @@ class Controller({{#apiInfo}}{{#apis}}{{#operations}}{{classVarName}}: {{classna
       }
    }
 
-     def swaggerRoute: Route =
-         pathPrefix("{{projectName}}" / interfaceVersion / "swagger") {
-           getFromResourceDirectory("swagger-ui") ~ getFromFile(tmpFile)
-     }
+   def swaggerRoute: Route =
+       pathPrefix("{{projectName}}" / interfaceVersion / "swagger") {
+         getFromResourceDirectory("swagger-ui") ~ getFromFile(tmpFile)
```

Comment:
I think you'd also rename the inner folder of `swagger-ui` from `swagger-ui` to `docs`. See [this](https://github.com/pagopa/pdnd-interop-uservice-authorization-process/pull/62/files), for example.